### PR TITLE
Лудина Д.А. 381806-3. ЛР3

### PR DESCRIPTION
**Pull request by dasha-ludina**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [ ] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [x] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [x] Get two approve from your fellow students
- [x] Take a review from one of the administrators

### CODE REVIEW

**Comment by dasha-ludina** [comment]
@shekotilova-yuliya @sashatyurmina посмотрите пожалуйста

File: modules/mortgage/application/mortgage_app.cpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Copyright 2021 Ludina Daria
+
+#include <iostream>
+#include <string>
+#include "include/mortgage_app.h"
```
Comment (shekotilova-yuliya) [inline modules/mortgage/application/mortgage_app.cpp:5]: необходима пустая строка

File: modules/mortgage/src/mortgage_app.cpp [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Ludina Daria
+
+#include <string>
+#include <sstream>
+#include "include/mortgage.h"
+#include "include/mortgage_app.h"
```
Comment (shekotilova-yuliya) [inline modules/mortgage/src/mortgage_app.cpp:6]: необходима пустая строка

File: modules/mortgage/test/test_mortgage_app.cpp [hunk 3]
```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Ludina Daria
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <vector>
+#include <algorithm>
+#include <functional>
+#include <iterator>
+#include "include/mortgage_app.h"
```
Comment (shekotilova-yuliya) [inline modules/mortgage/test/test_mortgage_app.cpp:10]: необходима пустая строка

**Review by shekotilova-yuliya — APPROVED** [review]

**Review by sashatyurmina — APPROVED** [review]

File: modules/mortgage/test/test_mortgage_app.cpp [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    const char** argv = &options.front();
+    int argc = static_cast<int>(args_.size()) + 1;
+
+    output_ = app_(argc, argv);
+  }
+
+  void Assert(std::string expected) {
+    EXPECT_TRUE(RE::PartialMatch(output_, RE(expected)));
```
Comment (streltsova-yana) [inline modules/mortgage/test/test_mortgage_app.cpp:34]:
```suggestion
    ASSERT_TRUE(RE::PartialMatch(output_, RE(expected)));
```

**Review by streltsova-yana — CHANGES_REQUESTED** [review]

**Review by streltsova-yana — APPROVED** [review]

**Pull request merged**